stdlib/LinearAlgebra/test/triangular.jl

```diff
@@ -187,6 +187,11 @@ for elty1 in (Float32, Float64, BigFloat, ComplexF32, ComplexF64, Complex{BigFlo
         @test imag(A1) == imag(Matrix(A1))
         @test abs.(A1) == abs.(Matrix(A1))
 
+        # zero
+        if A1 isa UpperTriangular || A1 isa LowerTriangular
+            @test zero(A1) == zero(Matrix(A1))
```

Comment:
```suggestion
            @test zero(A1) == zero(parent(A1))
```

... to save one copy?

Reply:
Yes, there are many copies in this testset. I'll submit another PR removing the allocations.